removed root of trust terminology fixes #395 (#397)
ietf-rats-wg/architecture: draft-ietf-rats-architecture.md

compromised devices from getting access to the network and potentially
harming others.

> Typically, a trustworthy solution starts with a specific component (sometimes referred to as the root of trust) that often

Reviewer: ```suggestion
Typically, a trustworthy solution starts with a specific component (sometimes referred to as a root of trust) that often
```

provides trustworthy device identity, and performs a series of operations that enables trustworthiness appraisals for other components.

> Such components perform operations that help determine the trustworthiness of yet other components,
> such as by collecting, protecting or signing measurements.

Reviewer 2: ```suggestion
Such components perform operations that help determine the trustworthiness of yet other components,
by collecting, protecting or signing measurements.
```

> such as collecting, protecting or signing measurements.

Reviewer: ```suggestion
such as by collecting, protecting or signing measurements.
```

> Measurements that have been signed by such components comprise Evidence that either supports or contradicts a claim of trustworthiness.

Reviewer 2: ```suggestion
Measurements that have been signed by such components comprise Evidence that when evaluated either supports or refutes a claim of trustworthiness.
```

> Measurements that have been signed by a trusted component is Evidence that either supports or contradicts a claim of trustworthiness.

Reviewer: ```suggestion
Measurements that have been signed by a trusted component comprises Evidence that either supports or contradicts a claim of trustworthiness.
```

> Measurements that have been signed by a trusted component comprises Evidence that either supports or contradicts a claim of trustworthiness.

Reviewer: ```suggestion
Measurements that have been signed by such components comprise Evidence that either supports or contradicts a claim of trustworthiness.
```

> Measurements can describe a variety of component attributes, such as hardware, firmware, BIOS, software, etc.

Reviewer 2: ```suggestion
Measurements can describe a variety of attributes of system components, such as hardware, firmware, BIOS, software, etc.
```

By definition, the Attester role generates Evidence.
An Attester may consist of one or
more nested environments (layers).

> The bottom layer of an Attester has an Attesting Environment that is implicitly trusted, typically because it is designed to be immutable or difficult to modify by malicious code.

Reviewer: ```suggestion
The bottom layer of an Attester has an Attesting Environment that is typically designed to be immutable or difficult to modify by malicious code.
```

In order to appraise Evidence generated by an Attester, the Verifier needs to trust various layers, including the bottom Attesting Environment.
Trust in the Attester's layers, including the bottom layer, can be established in various ways as discussed in {{verifier}}.

> In layered attestation, an implicitly trusted Attesting Environment is the initial layer.

Reviewer: If you have endorsements, then it's not "implicitly" trusted.

> In layered attestation, an implicitly trusted Attesting Environment is the initial layer.
> Claims can be collected from or about each layer beginning with an initial layer.

Reviewer: ```suggestion
In layered attestation, Claims can be collected from or about each layer beginning with an initial layer.
```

The corresponding Claims can be structured in a nested fashion that reflects the nesting of the Attester's layers.
Normally, Claims are not self-asserted, rather a previous layer acts as the Attesting Environment for the next layer.

> Claims about an implicitly trusted layer typically are asserted by an Endorser.

Reviewer: ```suggestion
Claims about an initial layer typically are asserted by an Endorser.
```

   1. The key material used to authenticate and integrity protect the conveyance channel is trusted by the Verifier to speak for the Attesting Environment(s) that collected Claims about the Target Environment(s).
   2. All unprotected Evidence that is conveyed is supplied exclusively by the Attesting Environment that has the key material that protects the conveyance channel

>    3. A trusted environment protects the conveyance channel key material which may depend on other Attesting Environments with equivalent strength protections.

Reviewer 2: ```suggestion
   3. A trusted environment protects the conveyance channel's key material which may depend on other Attesting Environments with equivalent strength protections.
```
